crates/sargon/src/radix_connect/wallet_interaction/dapp_wallet_interaction/wallet_to_dapp/success_response/pre_authorization/subintent_response.rs

```diff
@@ -1,35 +1,73 @@
 use crate::prelude::*;
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
 pub struct WalletToDappInteractionSubintentResponseItem {
-    /// A hex encoded signed partial transaction.
-    #[serde(rename = "signedPartialTransaction")]
-    pub encoded_signed_partial_transaction: String,
+    /// A signed subintent
+    pub signed_subintent: SignedSubintent,
```

Comment:
This is he change you had requested in https://github.com/radixdlt/sargon/pull/261 @CyonAlexRDX 

I wasn't able to implement it using `#[serde_as]`, and had to add custom `Serialize`/`Deserialize` implementations. Let me know if you know of an easier way of doing so.

Reply:
is `SignedSubintent` Serde? if not, that is why you could not use derive here. If it is not, why not? is out our type?

Reply:
ah sorry, I reviewed a bit hastily. I see now... but still the custom Serde should go on `SignedSubintent` and not `WalletToDappInteractionSubintentResponseItem` (which **can** derive Serde then, since it only consists of serde fields)

Reply:
if we add the custom Serialize/Deserialize inside `SignedSubintent` (rather than here), then every usage of it will also include the `subintentHash` fields (which isn't really necessary). 
To avoid this, we would need to add custom serialization in both places (and include the `subintentHash` here), which seems a bit unnecessary.

Given we don't need serialization of `SignedSubintent`, I guess it is fine to keep as is, but let me know what you think.